Index: .github/pull_request_template.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>## \uD83D\uDCA1 문제 풀이 요약\n\n### 문제 요구사항\n\n-\n\n### 접근 방법\n\n-\n\n### 최종 풀이 설명\n\n-\n---\n## ⚙\uFE0F 성능 최적화\n\n-\n---\n\n## \uD83D\uDCCC 체크리스트\n- [ ] 브랜치 확인!!! (절대 main으로 push하지 않기)\n- [ ] 다른 사람 코드 건드리지 않기!!
===================================================================
diff --git a/.github/pull_request_template.md b/.github/pull_request_template.md
--- a/.github/pull_request_template.md	(revision 023b7964c16cb8ec8ffebd08382398e85042c599)
+++ b/.github/pull_request_template.md	(date 1757291410060)
@@ -1,6 +1,7 @@
 ## 💡 문제 풀이 요약
 
-### 문제 요구사항
+
+###  문제 요구사항
 
 -
 
